pkg/runtime/runtime.go

```diff
@@ -33,9 +33,10 @@ type Interface interface {
 }
 
 type Metadata struct {
-	Version string `json:"version"`
-	Arch    string `json:"arch"`
-	Variant string `json:"variant"`
+	Version     string `json:"version"`
+	Arch        string `json:"arch"`
+	Variant     string `json:"variant"`
+	KubeVersion string `json:"kubeVersion"`
```

Comment:
What difference to Version?

Reply:
use KubeVersion to save version constraint.